script.js

```diff
@@ -123,10 +272,11 @@ function saveBtnClicked() {
     picture_url: pictureUrlValue,
     ingredients: ingredientsList,
     description: descriptionValue,
+    category: categoryValue,
```

Comment:
If you'd align the names of keys and values to be identical, you could write this in a more condensed way. Eg.
```suggestion
    picture_url,
    ingredients,
    description,
    category,
```

Reply:
I can understand what you mean! can you elaborate more or suggest articles for more info ?

Reply:
Yes, if your variables were named as the keys in the object, eg:
```
  let title = document.getElementById("title").value;
  let picture_url = document.getElementById("picture").value;
  let description = document.getElementById("descriptionArea").value;
  let category = document.getElementById("category").value;
  let ingredients = [];
```

then you would be able to write your object definition like this:
```
  const newRecipe = {
    id: recipeCounter,
    title,
    picture_url,
    ingredients,
    description,
    category
};
```

When key and value are identical you don't need to write:
```
category: category,
```
you can just write 
```
category,
```